src/components/dapp-staking/dapp/ProjectDetails.vue

```diff
@@ -113,52 +125,55 @@
                 <span class="text--tooltip">{{ $t('common.twitter') }}</span>
               </q-tooltip>
             </button>
+            <button v-if="it.type === CommunityType.Reddit" class="box--share btn--primary">
+              <div class="icon--social">
+                <a :href="it.handle" target="_blank" rel="noopener noreferrer">
+                  <astar-icon-base :icon-name="CommunityType.Reddit">
+                    <astar-icon-reddit />
+                  </astar-icon-base>
+                </a>
+              </div>
+              <q-tooltip>
+                <span class="text--tooltip">{{ $t('common.reddit') }}</span>
+              </q-tooltip>
+            </button>
+            <button class="box--share btn--primary">
+              <div class="icon--social">
+                <a :href="it.handle" target="_blank" rel="noopener noreferrer">
```

Comment:
It looks like the facebook icon navigate to twitter
![image](https://user-images.githubusercontent.com/92044428/201337426-cfbb7601-6342-49a3-aa00-185e6570cebe.png)

Reply:
And this navigates to YouTube
![image](https://user-images.githubusercontent.com/92044428/201337739-f059b120-a81b-4e3c-b71b-ad00cfbb887d.png)

Reply:
it's related with above issue